ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-device-documentation.j2

```diff
@@ -32,6 +32,8 @@
 {% include 'documentation/management-api-gnmi.j2' %}
 {## Management CVX #}
 {% include 'documentation/management-cvx.j2' %}
+{## CVX #}
```

Comment:
Watch out for the headings in the documentation. The CVX doc is creating a level-1 heading, but it is inserted between templates that are part of the level-1 heading "Management" above. Please move CVX below the Management section or make it a level-2 heading.